src/components/KTableData/KTableData.vue

```diff
@@ -275,39 +272,43 @@ const tablePaginationAttributes = computed((): TablePaginationAttributes => ({
   currentPage: page.value,
   offsetPreviousButtonDisabled: !previousOffset.value,
   offsetNextButtonDisabled: !nextOffset.value || !hasNextPage.value,
-  ...props.paginationAttributes,
+  ...paginationAttributes,
 }))
 
-function isTableColumnSlotName(slot: string): slot is TableColumnSlotName {
+const isTableColumnSlotName = (slot: string): slot is TableColumnSlotName<ColumnKey> => {
```

Comment:
nit: more comments in the code as to what different constants are for may be more valuable now that the types are more complex

Reply:
Added.